plugins/configs/mzML.LCMS_to_MetaboliteAtlas2_MAFileInfo.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Matt has final say.  But if it is more consistent naming I would think he
would be for it.

On Thu, May 7, 2015 at 11:32 AM, Roman Sutormin <notifications@github.com>
wrote:

> In plugins/configs/mzML.LCMS_to_MetaboliteAtlas2_MAFileInfo.json
> <https://github.com/kbase/transform/pull/151#discussion_r29881731>:
>
> > @@ -0,0 +1,84 @@
> > +{
>
> Is it possible to rename this file from
> mzML.LCMS_to_MetaboliteAtlas2_MAFileInfo.json
> into
> mzML.LCMS_to_MetaboliteAtlas2.MAFileInfo.json
> ?
> It's not so much required. It's just keeping naming system consistent
> (take a look at other config file names).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kbase/transform/pull/151/files#r29881731>.
>